ebuild_generator/ebuild.py

```diff
@@ -176,7 +176,7 @@ def get_ebuild_text(self, distributor, license_text, die_msg=None):
                 ret += ')"\n'
             else:
                 ret += "LICENSE=\""
-                ret += get_license(self.upstream_license) + "\"\n\n"
+                ret += get_license(self.upstream_license.replace(' ', '')) + "\"\n\n"
```

Comment:
I'd prefer the get_license function have this kind of line for the default return case (the one selected when none of the regexes match).